pages/docs/reason-compiler/latest/better-data-structures-printing-debug-mode.mdx

```diff
@@ -22,7 +22,7 @@ To remediate this, we introduce a flag for `bsc` (the BuckleScript compiler), `-
 
 ## Usage
 
-- Add `"bsc-flags": ["-bs-g"]` to your `bsconfig.json`.
+- Add `"bsc-flags": ["-bs-g"]` to your `rescript.json`.
 - In the BuckleScript/Reason file you'd like to debug, add `[%%debugger.chrome]` at the top \*.
```

Comment:
Unrelated, but this feels like quite outdated content...?

Reply:
right, I reverted that occurence